test/parallel/test-crypto-key-objects.js

```diff
@@ -33,18 +33,6 @@ const publicDsa = fixtures.readKey('dsa_public_1025.pem', 'ascii');
 const privateDsa = fixtures.readKey('dsa_private_encrypted_1025.pem',
                                     'ascii');
 
-{
-  // Attempting to create an empty key should throw.
-  assert.throws(() => {
-    createSecretKey(Buffer.alloc(0));
```

Comment:
Could you elaborate why this is necessary? Some algorithm requiring support for 0-length keys?

Reply:
- HKDF IKM is allowed as 0-length
- PBKDF2 password is allowed as 0-length
- The underlying CryptoKey structure is KeyObject

Reply:
Thank you!